lib/pymedphys/_experimental/chartchecks/compare.py

```diff
@@ -229,3 +237,20 @@ def weekly_check_color_results(val):
         return ["background-color: #C1FFC1"] * len(val)
     else:
         return ["background-color: #EE6363"] * len(val)
+
+
+def specific_patient_weekly_check_color_results(val):
```

Comment:
Not a must... but the rest of the code base uses british english... might be some value to stick to the standard of the code-base so that people who want to call PyMedPhys related things can stick to one standard. -- referring to "color" vs "colour".